### DISCUSSION TITLE
How to get top,left,right and bottom pixels of images in imagemagick

### DISCUSSION THREAD

**Discussion by atfsolace** [ImageMagick/ImageMagick#2464] [Help]
Hi,
I have a bunch of images and I want to get the top,left,right and bottom pixels of images.

So How to find the top,left,right and bottom pixels (including white background) of images from staring point of the image to the inner border of actual image.

Example image:
![image](https://user-images.githubusercontent.com/70207338/91172540-fdf29480-e6f9-11ea-8691-87f268dd1c18.png)

Original Image:
![image](https://user-images.githubusercontent.com/70207338/91173099-e7990880-e6fa-11ea-8765-5d47ac589f6f.png)

Thank you for any suggestions

**Comment by snibgo** [comment]
What version of IM, on what platform?

I don't understand what you want.

From an image, we can make a 2x2 pixel image from the corners of the input using "-chop", like this:
```
magick in.png -gravity center -chop %[fx:w-2]x%[fx:h-2]+0+0 out.png
```
We can crop the input before "-chop" if we want.

**Reply by atfsolace** [reply]
Hi thank you for your valuable reply.
I am using Version: ImageMagick 7.0.10-24 Q16 x64 2020-07-18 http://www.imagemagick.org on Windows platform.

I want to get the top,left,right and bottom pixels values from images, Need a ratio about how much image is left and right top and bottom to inner image or how much space from left and right top and bottom to inner border of image

Inner image:
![image](https://user-images.githubusercontent.com/70207338/91176068-61cb8c00-e6ff-11ea-86d0-86d61909b570.png)

Left side part of image:
![image](https://user-images.githubusercontent.com/70207338/91176134-76a81f80-e6ff-11ea-8a34-8c65d9cbf948.png)

Top side part of image:
![image](https://user-images.githubusercontent.com/70207338/91176180-858ed200-e6ff-11ea-808f-ef522489b8ac.png)

Right side part of image:
![image](https://user-images.githubusercontent.com/70207338/91176230-95a6b180-e6ff-11ea-87c2-cabcb3dc4706.png)

Bottom side part of image:
![image](https://user-images.githubusercontent.com/70207338/91176277-a820eb00-e6ff-11ea-9c88-174bd8804f67.png)

Thank you.
